concent_api/core/message_handlers.py

```diff
@@ -490,79 +540,80 @@ def handle_send_force_payment(request, client_message: message.concents.ForcePay
             subtask_results_accepted.task_to_compute.provider_public_key,
         )
 
-    task_to_compute = client_message.subtask_results_accepted_list[0].task_to_compute
```

Comment:
Why was this removed ? It was added on purpose, to shorten usages and make code cleaner. Please add it back and apply where possible in this handler.

Reply:
`task_to_compute` restored and repaird